payments-core/src/main/java/com/stripe/android/payments/core/authentication/Stripe3ds2TransactionViewModel.kt

```diff
@@ -0,0 +1,15 @@
+package com.stripe.android.payments.core.authentication
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+
+internal class Stripe3ds2TransactionViewModel : ViewModel() {
+    // TODO(mshafrir-stripe): move logic from Stripe3ds2Authenticator into ViewModel
+}
+
+internal class Stripe3ds2TransactionViewModelFactory : ViewModelProvider.Factory {
```

Comment:
move it inside the ViewModel class?

Reply:
I like having them separate for readability